specificatie/genereervariant/openapi.yaml

```diff
@@ -198,13 +198,13 @@ paths:
       tags:
       - Adres
       summary: vindt adressen
-      description: Vind een actueel adres met de identificatie van het zoekresultaat
-        uit de operatie get /adressen/zoek, de pandidentificatie of de adresseerbaarobjectidentificatie.
-        Gebruik de fields-parameter als je alleen specifieke velden in het antwoord
-        wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/fields.feature).
-        Gebruik de expand-parameter als je het antwoord wil uitbreiden met (delen
-        van) de gerelateerde resources nummeraanduiding, woonplaats en openbare ruimte,
-        zie [functionele specificaties expand-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/expand.feature).
+      description: "Vind een actueel adres met de identificatie van het zoekresultaat\
+        \ uit de operatie get /adressen/zoek, de pandidentificatie of de adresseerbaarobjectidentificatie.\
+        \ \nGebruik de fields-parameter als je alleen specifieke velden in het antwoord\
+        \ wil zien,  zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/fields.feature).\
+        \ \nGebruik de expand-parameter als je het antwoord wil uitbreiden met (delen\
+        \ van) de gerelateerde resources nummeraanduiding, woonplaats, openbare ruimte\
+        \ en adresseerbaarobject,  zie [functionele specificaties expand-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/expand.feature)."
       operationId: raadpleegAdressen
```

Comment:
Er worden nu ook backslashes toegevoegd aan de description in de genereervariant. Is daar ook een reden voor ? Ik heb overigens geen flauw idee of dit een probleem is.

Reply:
Dat wordt gedaan door de resolver, overigens maakt het dan niet uit of je >- of | gebruikt, in beide gevallen komen er backslashes voor te staan.